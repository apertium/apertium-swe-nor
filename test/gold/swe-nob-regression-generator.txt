[2GDnjhHKwf6J]
Mange i landet er smittet av koronaviruset. [/option]
[/2GDnjhHKwf6J]
[4ezRqmPuJ3lS]
har blitt oppført [/option]
[/4ezRqmPuJ3lS]
[BGKetWFycO5a]
Det er umulig å reise ut av Praha til andre plasser i landet. [/option]
[/BGKetWFycO5a]
[BGSXM-VaHxKz]
som ikke har blitt oppført [/option]
[/BGSXM-VaHxKz]
[BcOHchZgEltn]
mer overskuelig. [/option]
[/BcOHchZgEltn]
[EM8lSAP-warx]
Jeg fikk også en ny kusine i år som jeg nesten ikke har sett. [/option]
[/EM8lSAP-warx]
[Ep5NrTWjxfXF]
Tittel: Anna får ikke reise fra byen hun bor i. [/option]
[/Ep5NrTWjxfXF]
[FGbYle80JvRe]
ham. [/option]
[/FGbYle80JvRe]
[I1a7gtD3zHRi]
har funnes [/option]
[/I1a7gtD3zHRi]
[URVJjX_lB38B]
mest overskuelige. [/option]
[/URVJjX_lB38B]
[V01qA6qi0kKt]
Ingress: Anna er bare sammen med vennene sine på Teams nå for tiden. [/option]
[/V01qA6qi0kKt]
[W-fqMeU--uw4]
som har funnes [/option]
[/W-fqMeU--uw4]
[Y69rKruaEuON]
Jeg har åler i min luftputebåt. [/option]
[/Y69rKruaEuON]
[_C4fXLa-MSE3]
Hvordan har dette siste året vært? [/option]
[/_C4fXLa-MSE3]
[cN2TnhvQvwvm]
mest overskuelig. [/option]
[/cN2TnhvQvwvm]
[dMemZE94Geg0]
Pannekaker og lasagne. [/option]
[/dMemZE94Geg0]
[eLrB2F8Y26uc]
søkende der evnen til å utnytte [/option]
[/eLrB2F8Y26uc]
[gqecOW-8qUFZ]
har blitt oppført. [/option]
[/gqecOW-8qUFZ]
[iVpJjgVw2Siq]
Det er alle reglene, at vi må bruke munnbind, og at jeg ikke kan gå hvor jeg vil. [/option]
[/iVpJjgVw2Siq]
[igT2HXzyiwds]
har funnes. [/option]
[/igT2HXzyiwds]
[jONRw3Es7Fmy]
de parter som har søknader som [/option]
[/jONRw3Es7Fmy]
[xr87sIaNgHKM]
Tsjekkia har blitt rammet hardt av koronaviruset. [/option]
[/xr87sIaNgHKM]
[yaT9Y83vKE5G]
Det har vært trist! [/option]
[/yaT9Y83vKE5G]
[zTuZzFUZoMgA]
ham [/option]
[/zTuZzFUZoMgA]
