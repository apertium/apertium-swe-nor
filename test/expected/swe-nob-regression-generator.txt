[0tkleIoBBRxj#0] mest overskuelig
[/0tkleIoBBRxj]
[4ezRqmPuJ3lS#0] har blitt oppført
[/4ezRqmPuJ3lS]
[BGSXM-VaHxKz#0] som ikke har blitt oppført
[/BGSXM-VaHxKz]
[I1a7gtD3zHRi#0] har blitt funnet
[/I1a7gtD3zHRi]
[W-fqMeU--uw4#0] som har blitt funnet
[/W-fqMeU--uw4]
[eLrB2F8Y26uc#0] søkende der evnen til å utnytte
[/eLrB2F8Y26uc]
[hqvcBLgqrUlV#0] mest overskuelige
[/hqvcBLgqrUlV]
[i9_os4amzrP1#0] mer overskuelig
[/i9_os4amzrP1]
[jONRw3Es7Fmy#0] de parter som har søknader som
[/jONRw3Es7Fmy]
[tMKlZei_zLhv#0] Jeg har åler i min luftputebåt
[/tMKlZei_zLhv]
[zTuZzFUZoMgA#0] ham
[/zTuZzFUZoMgA]
